unit_converter/__init__.py

```diff
@@ -0,0 +1,460 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import annotations
+
+import json
+import re
+import traceback
+from datetime import datetime
+from pathlib import Path
+from typing import Any
+from urllib.request import urlopen
+
+import albert
+import inflect
+import pint
+
+default_trigger = "convert "
+synopsis = "<amount> <from_unit> to <to_unit>"
+
+__doc__ = f"""
+Extension for converting units of length, mass, speed, temperature, time,
+current, luminosity, printing measurements, molecular substance, currency, and more
+
+Synopsis: {default_trigger}{synopsis}
+
+Examples:
+`{default_trigger}180 minutes to hrs`
+`{default_trigger}100 km to miles`
+`{default_trigger}88 mph to kph`
+`{default_trigger}32 degrees F to C`
+`{default_trigger}3.14159 rad to degrees`
+`{default_trigger}100 USD to EUR`
+"""
+
+md_iid = "0.5"
+md_version = "1.2"
+md_name = "Unit Converter"
+md_description = "Convert length, mass, temperature, time, currency, and more"
```

Comment:
Please keep this brief.  Convert units is enough. Long string make problems in the ui